src/lib/core/ripple/ripple.ts

```diff
@@ -102,11 +107,11 @@ export class MdRipple implements OnChanges, OnDestroy {
 
   /** Ripple configuration from the directive's input values. */
   get rippleConfig(): RippleConfig {
-    return {
+    return extendObject({
       centered: this.centered,
       speedFactor: this.speedFactor,
       radius: this.radius,
       color: this.color
-    };
+    }, this._globalRippleConfig);
```

Comment:
That's what I was trying to avoid with the config. The config is for the single ripple references and doing stuff like below is just not a good API.

```ts
ripple.launch(0, 0, { disabled: true });
```

> Why would you want to launch a ripple that is disabled?

You are not disabling the single ripple-ref, you are disabling the whole ripple triggers.